.travis.yml

```diff
@@ -9,9 +9,10 @@ cache:
   directories:
     - $HOME/.cache/go-build
 install:
-  - ./bin/setup-travis
-script:
-  - cd $HOME/gopath/src/github.com/teamwork/guru
-  - ./bin/test -race ./...
-  - ./bin/coverage
-  - ./bin/lint ./...
```

Comment:
linting's gone now, maybe add golangci-lint? 🤔

Reply:
Yeah could add it I guess, since it's such a small package with no real changes I didn't really bother with it 😅 Been using [a fork](https://github.com/zgoat/guru) which changes a few other things, so to be honest I don't really care all that much, feel free to amend or just close this :-)